src/clashes.go

```diff
@@ -20,6 +20,13 @@ import (
 	"strings"
 )
 
+type FixClashesMode uint8
+
+const (
+	FixClashesRename FixClashesMode = iota
```

Comment:
Let's make this `1 + iota`, so that when we  can catch the
zero/uninitialized value, and reject it.